core/state_transition_ovm.go

```diff
@@ -83,14 +83,15 @@ func asOvmMessage(tx *types.Transaction, signer types.Signer) (Message, error) {
 	}
 
 	// ovmTODO: Is this still necessary?
```

Comment:
We should revisit this TODO

Reply:
Ah hmm...  I am not 100% sure but I think it might still be useful to keep this logic because we don't have a vrs if the nil is used as sender